scripts/periodic/410.pkg-audit.in

```diff
@@ -156,6 +157,17 @@ audit_pkgs_all() {
 	esac
 
 	for j in $jails ; do
+		# ignore some jails
+		# we iterate to get exact matches because we want substring matches
+		# foo should not match foo.bar
+		for ignore in $security_status_pkgaudit_jails_ignore ; do
```

Comment:
all the scripts uses ${var} and not $var

Reply:
Fixed.